mobile/src/reducers/errorHandling/error.reducer.ts

```diff
@@ -0,0 +1,25 @@
+import { Reducer, combineReducers } from 'redux';
+import { OrganizationActions } from '../organization/organization.action';
+import { Alert } from 'react-native';
+
+var showAlert: boolean = false;
+function getAlert(message: string) {
+    Alert.alert('Error', message);
```

Comment:
this code should not be here, it's supposed to be in tsx. I'll take a minute later, maybe we *can* put it in epics as you wanted at the beginning... but definitely not in reducers